package.json

```diff
@@ -56,7 +56,57 @@
         "url": "https://github.com/zowe/api-layer/issues"
     },
     "homepage": "https://github.com/zowe/api-layer#readme",
-    "dependencies": {},
+    "dependencies": {
```

Comment:
this is the wrong package.json, most of the findings are in `catalog-ui/frontend`. This may be confusing the black duck scanner